CHANGELOG.md

```diff
@@ -1,11 +1,115 @@
 # CHANGELOG
 
+## [Unreleased]
+
+Add an entry to the unreleased section whenever merging a PR to main that is not targeted at a specific release. These entries will eventually be included in a release.
+
+## v.1.3.0
+
+Date: May 12th, 2023
+
+Unlike prior releases, the ICS v1.3.0 release will be based on the main branch. v1.3.0 will contain all the accumulated PRs from the various releases below, along with other PRs that were merged, but not released to production. After v1.3.0, we plan to revamp release practices, and how we modularize the repo for consumer/provider.
+
+Some PRs from v1.3.0 may reappear from other releases below. This is due to the fact that ICS v1.1.0 deviates from the commit ordering of the main branch, and other releases thereafter are based on v1.1.0.
+
+### High level changes included in v1.3.0
+
+* MVP for standalone to consumer changeover, see [EPIC](https://github.com/cosmos/interchain-security/issues/756)
+* MVP for soft opt out, see [EPIC](https://github.com/cosmos/interchain-security/issues/851)
+* Various fixes, critical and non-critical
+* Docs updates which should not affect production code
+
+## PRs included in v1.3.0
+
+* (fix) Prevent denom DOS [#931](https://github.com/cosmos/interchain-security/pull/931)
+* (fix) multisig for assigning consumer key, use json [#916](https://github.com/cosmos/interchain-security/pull/916)
+* (feat) Add warnings when provider unbonding is shorter than consumer unbonding [#858](https://github.com/cosmos/interchain-security/pull/858)
+* (chore) use go 1.19 [#899](https://github.com/cosmos/interchain-security/pull/899)
+* (feat) Standalone to consumer changeover - recycle existing transfer channel [#832](https://github.com/cosmos/interchain-security/pull/832)
+* (deps) Bump IBC [862](https://github.com/cosmos/interchain-security/pull/862)
+* (chore) Use go 1.20 [#840](https://github.com/cosmos/interchain-security/pull/840)
```

Comment:
Combine this and https://github.com/cosmos/interchain-security/pull/899 into one entry.